### PR TITLE
[8.8] Vulnerability management integration docs

### LINKED ISSUE #3158: 8.8 Vulnerability Management documentation
In 8.8, we are introducing a new integration which scans AWS EC2 workloads for vulnerabilities: Elastic Vulnerability Management for Cloud (Cloud Native Vulnerability Management). This is being considered a security posture capability.

We need to add a section within the Cloud Native Security docs for vuln management, with at least two pages to start: an overview page, and a getting started page with deployment instructions.

### PR DESCRIPTION
**Pull request by benironside**
Fixes #3158 

Adds pages for the Cloud Native Vulnerability Management (CNVM) feature / Elastic Vulnerability Management for Cloud (EVMC) integration.

3 new pages: [Preview](https://security-docs_3162.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-overview.html)

### CODE REVIEW

File: docs/cloud-native-security/vuln-management-findings.asciidoc
Comment (nick-alayil) [inline docs/cloud-native-security/vuln-management-findings.asciidoc:4]: Replace Elastic Vulnerability Management for Cloud (EVMC) integration to Cloud native vulnerability management (CNVM) integration.

File: docs/cloud-native-security/vuln-management-findings.asciidoc
Comment (nick-alayil) [inline docs/cloud-native-security/vuln-management-findings.asciidoc:34]: Instead of newer version, it's better to say fix version.

**Review by nick-alayil — APPROVED** [review]
LGTM. I've updated my minor comments.

File: docs/cloud-native-security/vuln-management-overview.asciidoc
Comment (nastasha-solomon) [inline docs/cloud-native-security/vuln-management-overview.asciidoc:28]:
```suggestion
The scanning process begins immediately upon deployment, then repeats every twenty-four hours. After each scan, the integration sends the discovered vulnerabilities to {es}, where they appear in the **Vulnerabilities** tab of the <<vuln-management-findings, Findings page>>.
```

**Review by nastasha-solomon — COMMENTED** [review]
Very clear and well-written instructions and explanations, @benironside! I left some suggestions and questions for your consideration but overall, everything looks great!

File: docs/cloud-native-security/vuln-management-findings.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-findings.asciidoc:33]: Yes down the line I think we'll need to add more options here, but you're right there's no reason to make it a list until then :)

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:11]: good question, I'm following up on this but for now going to leave this comment unresolved

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:29]: Good catch. In the BC, you need to enable "Display beta integrations" when searching for integrations. Once you do that, the integration will appear in search. After release, users won't have to do this

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:39]: https://helpcenter.veeam.com/docs/styleguide/tw/log_in_to.html Think the original is correct

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:50]: The cloud formation template appears in AWS, so I think it's accurate to say go back to Kibana

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:52]: changing the other fields would mess up the auth/identity and completely stop the integration from working, I'll say a little more about that 👍

File: docs/cloud-native-security/vuln-management-overview.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-overview.asciidoc:6]: I think it's fairly standard to just refer to it as "IaC" rather than "an IaC approach" 😃

**Review by jmikell821 — COMMENTED** [review]
Hey @benironside, left some suggestions for your review. I think you could add some more screenshots in a few places. If you have one of AWS, that'd be great, but if not, understood. I'd like us to take another pass once feedback is merged. Thanks!

File: docs/cloud-native-security/vuln-management-overview.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-overview.asciidoc:30]: It would mean more VMs. I can just say that

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:39]: My thinking on making it a separate section is that you have to context switch to complete the steps in this section (need to switch tabs and sign in to AWS), so I figure it may help people keep track of where they are in the instructions when they switch back to the docs. Happy to streamline here if you prefer

File: docs/cloud-native-security/vuln-management-faq.asciidoc
Comment (benironside) [inline docs/cloud-native-security/vuln-management-faq.asciidoc:40]:
```suggestion
To run the CloudFormation template for CNVM onboarding, you need an AWS user account with permissions to perform the following actions: run CloudFormation templates, create IAM Roles and InstanceProfiles, and create EC2 SecurityGroups and Instances.
```

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc]: Sorry, I'm not clear on which tidbit you mean here Janeen. To clarify, the users won't need to check "Display beta integrations", that's only a requirement in the BC.

File: docs/cloud-native-security/vuln-management-overview.asciidoc
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-overview.asciidoc:14]: This needs to be a complete sentence, like the rest of the list. (I know the sidebar is called "Requirements" so this is implied, but this is more for consistency.)
```suggestion
* Requires {stack} and {agent} version 8.8 or higher.
```

File: docs/cloud-native-security/vuln-management-overview.asciidoc
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-overview.asciidoc:32]: Remove? If the plan is to eventually expand the page with use cases, just add the heading at that time, not now.

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-get-started.asciidoc:10]:
```suggestion
* Requires {stack} and {agent} version 8.8 or higher.
```

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (joepeeples) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:42]: I had the same thought as @jmikell821 at first, but I get @benironside's approach too. One idea is to make the three sections one procedure, with three main steps each with their own substeps:

> 1. Add the CNVM integration
>     a. In the Elastic Security app, go to...
>     b. Search for Cloud...
>     c. Click Add Cloud... 
>     d. etc.
> 
> 2. Sign in to the AWS management console
>     a. Open a new browser
>     b. Switch to the cloud region
> 
> 3. Run the CloudFormation template
>     a. Switch back...
>     b. Click Launch...

Or, revise the headings to indicate they're all part of the same overarching procedure (the user has to do them all, in a certain order; they're not like normal sections where you can skip around more freely):

> === Step 1: Add the CNVM integration
> === Step 2: Sign in to the AWS management console
> === Step 3: Run the CloudFormation template

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-get-started.asciidoc:56]: This screenshot feels unnecessary — not because it's not our UI, but because the user is doing almost nothing on this screen other than just click through (with one optional config change). Even if this were a Kibana UI, I'd rethink whether the user needs a huge screenshot (and whether we want to maintain it) just to click a single, pretty conspicuous button.

File: docs/cloud-native-security/images/cnvm-findings-page.png
Comment (joepeeples) [inline docs/cloud-native-security/images/cnvm-findings-page.png:1]: Crop out unnecessary UI sections (side nav, title bar, etc.). _Maybe_ keep the search bar, since it's referenced in the text below, but personally I'd crop it out too, because it's so ubiquitous I don't think a reader would need to see it in a screenshot in order to find it.

File: docs/cloud-native-security/vuln-management-faq.asciidoc
Comment (joepeeples) [inline docs/cloud-native-security/vuln-management-faq.asciidoc:40]: Customers probably need to know this _before_ they try to start running the template. Include this info as prerequisites or requirements on the "Get started with CNVM" page?

**Review by joepeeples — COMMENTED** [review]
Awesome content and effort here! Added just a few suggestions, comments. Thanks!

File: docs/cloud-native-security/vuln-management-get-started.asciidoc
Comment (benironside) [reply docs/cloud-native-security/vuln-management-get-started.asciidoc:56]: I tend to agree but I'll defer to @jmikell821 on this one, since I added the AWS image at her suggestion.

**Comment by nick-alayil** [comment]
@benironside with all the changes made so far. It looks awesome documentation and a great start for a brand new feature. Thank you @benironside and all the reviewers (@jmikell821 , @nastasha-solomon , @joepeeples ) 🙏 !!

**Comment by nick-alayil** [comment]
A few minor comments/questions:

In [set up CNVM for AWS](https://security-docs_3162.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-get-started.html#vuln-management-setup) section, in the highlighted important note, we're asking the reader to go [How CNVM works](https://security-docs_3162.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-overview.html#vuln-management-overview-how-it-works) but the text shown is Vulnerability Management overview as shown below. Shouldn't the text be ```How CNVM works```?
> For more information, refer to [Vulnerability Management overview](https://security-docs_3162.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-overview.html#vuln-management-overview-how-it-works).   

In KSPM and CSPM , FAQ page header is ```Frequently asked questions (FAQ)``` but in CNVM it's just FAQ. I believe for uniformity, we'll stick to the former.

Also in [CNVM FAQ](https://security-docs_3162.docs-preview.app.elstc.co/guide/en/security/master/vuln-management-faq.html), the answers are not starting in a newline like [CSPM FAQ](https://security-docs_3162.docs-preview.app.elstc.co/guide/en/security/master/cspm-security-posture-faq.html).

File: docs/cloud-native-security/vuln-management-overview.asciidoc
Comment (jmikell821) [reply docs/cloud-native-security/vuln-management-overview.asciidoc]: @benironside @joepeeples I think historically, we use `earlier` and `later` when referring to stack versions. I think this was a UX writing question at one point, but I don't think we should use `higher`. Of course I went down a rabbit hole and found we use various forms of this in our docs... 🙃

**Pull request merged**
